### PR TITLE
Feature/layout rework

### PR DESCRIPTION
**Pull request by nicklee**
Updates to move to a fixed left column layout. Removes a lot of dependencies on bootstrap

Based on https://github.com/wagtail/guide/pull/119

### CODE REVIEW

File: apps/frontend/static_src/scss/components/skip-link.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.skip-link {
+    position: absolute;
+    z-index: 4;
+    top: 0;
+    left: 0;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/skip-link.scss:5]:
```suggestion
    inset-inline-start: 0;
```

File: apps/frontend/static_src/scss/components/skip-link.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        padding: 0;
+        margin: 0;
+        list-style: none;
+    }
+
+    &__link {
+        display: block;
+        position: absolute;
+        top: -200px;
+        font-size: 14px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/skip-link.scss:17]: rem

File: apps/frontend/static_src/scss/components/skip-link.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        left: 0;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/skip-link.scss:18]:
```suggestion
        inset-inline-start: 0;
```

File: apps/frontend/static_src/scss/components/streamfield.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.streamfield {
+    h2 {
+        font-size: 30px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/streamfield.scss:3]: Here and below – all font sizes in rem

File: apps/frontend/static_src/scss/mixins.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    padding: 0 $gutter;
+    max-width: $site-width;
+    margin: 0 auto;
+
+    @include media-query(medium) {
+        padding: 0 ($gutter * 3);
+    }
+
+    @include media-query(large) {
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/mixins.scss:32]: Just a tad easier to read

```suggestion
        grid-template-columns: repeat(10, 1fr);
```

File: apps/frontend/static_src/scss/main.scss [hunk 6]
```diff
@@ -15,47 +16,34 @@
...
+@import "./components/header.scss";
+@import "./components/headings.scss";
+@import "./components/streamfield.scss";
+@import "./components/link-block.scss";
+@import "./components/toc.scss";
+@import "./components/dropdown-toggle.scss";
+@import "./components/search.scss";
+@import "./components/grid.scss";
+@import "./components/language-selector.scss";
+@import "./components/skip-link.scss";
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/main.scss:49]: For consistency, could we either always have the `.scss` suffix or never?

File: apps/frontend/static_src/scss/components/toc.scss [hunk 7]
```diff
@@ -0,0 +1,38 @@
+.toc {
+    margin-bottom: ($gutter * 2);
+
+    @include media-query(large) {
+        margin-bottom: ($gutter * 3);
+    }
+
+    &__heading {
+        font-size: 16px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/toc.scss:9]: rem

File: apps/frontend/static_src/scss/components/toc.scss [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+            align-items: center;
+            margin-bottom: ($gutter * 0.25);
+
+            &::before {
+                content: "";
+                display: inline-block;
+                height: 22px;
+                width: 22px;
+                margin-right: 10px;
+                background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj48cGF0aCBmaWxsPSIjMDA3RDdFIiBkPSJNMjgwLjIgMTUwLjJjLTcuMS02LjQtMTguMS04LTI1LjktNC4xcy0xNS4yIDEyLjQtMTUuMiAyMWwuMDAyIDU2TDE1MiAyMjRjLTEzLjIgMC0yNCAxMC44LTI0IDI0djE2YzAgMTMuMyAxMC44IDI0IDI0IDI0bDg4LS45djU2YzAgOS41MzEgNS42NTYgMTguMTYgMTQuMzggMjJhMjQuMDI1IDI0LjAyNSAwIDAgMCAyNS45MS00LjM3NWw5Ni04OC43NUMzODEuMiAyNjguMyAzODQgMjYxLjkgMzg0IDI1NS4yYy0uMzEzLTcuNzgxLTIuODc1LTEzLjI1LTcuODQ0LTE3Ljc1TDI4MC4yIDE1MC4yek0yNTYgMEMxMTQuNiAwIDAgMTE0LjYgMCAyNTZzMTE0LjYgMjU2IDI1NiAyNTYgMjU2LTExNC42IDI1Ni0yNTZTMzk3LjQgMCAyNTYgMHptMCA0NjRjLTExNC43IDAtMjA4LTkzLjMxLTIwOC0yMDhTMTQxLjMgNDggMjU2IDQ4czIwOCA5My4zMSAyMDggMjA4LTkzLjMgMjA4LTIwOCAyMDh6Ii8+PC9zdmc+");
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/toc.scss:34]: For a project like this I think it’s important we don’t hard-code base64 icons like this, which will make it hard for us to recover the actual image if we ever need to.

If the tooling isn’t doing this currently and it feels like a waste of time, just add a comment to explain which icon this is / which SVG file, and we’ll open an issue to set this up properly.

**Review by thibaudcolas — CHANGES_REQUESTED** [review]

File: apps/frontend/static_src/scss/components/streamfield.scss (hunk 4 above)
Comment (nicklee) [reply apps/frontend/static_src/scss/components/streamfield.scss:3]: Fixed in the font refactor MR

File: apps/frontend/static_src/scss/main.scss (hunk 6 above)
Comment (nicklee) [reply apps/frontend/static_src/scss/main.scss:49]: Fixed in bootstrap evaluation branch

File: apps/frontend/static_src/scss/components/toc.scss (hunk 8 above)
Comment (nicklee) [reply apps/frontend/static_src/scss/components/toc.scss:34]: Done in bootstrap eval

**Pull request merged**